CarShowRoom/cars/api/v1/urls.py

```diff
@@ -7,3 +8,16 @@
 router.register(r"car", CarViewSet)
 
 urlpatterns = router.urls
+
+urlpatterns += [
+    path(
+        "car/<int:pk>/managedealercars/",
```

Comment:
Nit: usually endpoint name is a corresponding model name. If endpoint is mixed (e.g a single model name cannot be specified), then you can combine into one using hyphens 

https://restfulapi.net/resource-naming/

Reply:
Okey, will check it